pkgs/desktops/gnome-3/core/tracker/default.nix

```diff
@@ -33,27 +34,19 @@ in stdenv.mkDerivation rec {
     "-Dsystemd_user_services=lib/systemd/user"
     # TODO: figure out wrapping unit tests, some of them fail on missing gsettings-desktop-schemas
     "-Dfunctional_tests=false"
+    "-Ddocs=true"
   ];
 
   patches = [
-    # Always generate tracker-sparql.h in time
-    (fetchurl {
-      url = https://gitlab.gnome.org/GNOME/tracker/commit/3cbfaa5b374e615098e60eb4430f108b642ebe76.diff;
-      sha256 = "0smavzvsglpghggrcl8sjflki13nh7pr0jl2yv6ymbf5hr1c4dws";
+    (substituteAll {
+      src = ./fix-paths.patch;
+      glib-dev = glib.dev;
```

Comment:
Pretty sure if this isn't a valid bash variable `glib_dev` it won't work.

Reply:
ah, cheers